src/sections/Community/Handbook/repository.js

```diff
@@ -101,7 +102,7 @@ const Repository = () => {
 
               <li>
                 <a href="https://github.com/service-mesh-patterns">
-                  <img
+                  <StaticImage
                     src="https://user-images.githubusercontent.com/85789734/134711787-5bf0aeaa-008c-4e2c-a81e-e05f3dcfb54c.png"
```

Comment:
Just use this [svg](https://github.com/layer5io/layer5/blob/master/src/assets/images/service-mesh-patterns/service-mesh-pattern.svg) here, that would be better. Reference the local path similar to other images above